diff --git a/params/ligation-set.toml b/params/ligation-set.toml
index 88c017607..77834a1d9 100644
--- a/params/ligation-set.toml
+++ b/params/ligation-set.toml
@@ -1,279 +1,257 @@
 [simple.center-ops]
-ligGroup =  'center-ops'
 samples = [":=", ":-", ":+", "<*", "<*>", "*>", "+:", "-:", "=:", "<***>", "<:", ":>", "*-", "-*", "*=", "=*", "*+", "+*"]
 desc = 'Vertically align some of the operators (like `*`) to the center position it is before or after a "center" operator (like `+`)'

 [simple.center-op-trigger-plus-minus-l]
-ligGroup = 'center-op-trigger-plus-minus-l'
 samples = ["*+", "*-", ":+", ":-", ".+", ".-"]
 desc = 'Plus (`+`) and Minus (`-`) will trigger other operator characters at left to be centered'

 [simple.center-op-trigger-plus-minus-r]
-ligGroup = 'center-op-trigger-plus-minus-r'
 samples = ["+*", "-*", "+:", "-:", "+.", "-."]
 desc = 'Plus (`+`) and Minus (`-`) will trigger other operator characters at right to be centered'

 [simple.center-op-trigger-equal-l]
-ligGroup = 'center-op-trigger-equal-l'
 samples = ["*=", ":=", ".="]
 desc = 'Equal (`=`) will trigger other operator characters at left to be centered'

 [simple.center-op-trigger-equal-r]
-ligGroup = 'center-op-trigger-equal-r'
 samples = ["=*", "=:", "=."]
 desc = 'Equal (`=`) will trigger other operator characters at right to be centered'

 [simple.center-op-trigger-bar-l]
-ligGroup = 'center-op-trigger-bar-l'
 samples = ["*|", ":|"]
 desc = 'Bars (`|`) will trigger other operator characters at left to be centered'

 [simple.center-op-trigger-bar-r]
-ligGroup = 'center-op-trigger-bar-r'
 samples = ["|*", "|:"]
 desc = 'Bars (`|`) will trigger other operator characters at right to be centered'

 [simple.center-op-trigger-angle-inside]
-ligGroup = 'center-op-trigger-angle-inside'
 samples = ["<:", "<:>", ":>", "<*", "<*>", "*>", "<.", "<.>", ".>"]
 desc = 'Less (`<`) and Greater (`>`) will trigger other operator characters at inside to be centered'

 [simple.center-op-trigger-angle-outside]
-ligGroup = 'center-op-trigger-angle-outside'
 samples = [">*", "*<"]
 desc = 'Less (`<`) and Greater (`>`) will trigger other operator characters at outside to be centered'

 [simple.center-op-influence-dot]
-ligGroup =  "center-op-influence-dot"
 samples = ["<.", "<.>", ".>"]
 desc = 'Treat dot (`.`) as operator and perform chained centering'

 [simple.center-op-influence-colon]
-ligGroup =  "center-op-influence-colon"
 samples = ["<:", "<:>", ":>", ":=", "=:", ":-", "-:"]
 desc = 'Treat colon (`:`) as operator and perform chained centering'



 [simple.arrow-l]
-ligGroup =  "arrow-l"
-samples = ["<--", "<---", "<<-", "<-", "<->", "<-->", "<--->", "<---->", "<=", "<==", "<===", "<<=", "<=>", "<==>", "<===>", "<====>", "<~~", "<~", "<~>", "<=<", "<-<"]
+samples = ["<--", "<---", "<<-", "<-", "<-->", "<--->", "<---->", "<=", "<==", "<===", "<<=", "<==>", "<===>", "<====>", "<~~", "<~", "<=<", "<-<"]
 desc = 'Enable ligation set that forms left-pointing arrows'

 [simple.arrow-r]
-ligGroup =  "arrow-r"
-samples = ["->", "->>", "-->", "--->", "<->", "<-->", "<--->", "<---->", "=>", "=>>", "==>", "===>", "<=>", "<==>", "<===>", "<====>", "<~>", "~>", "~~>", "-------->", ">=>", ">->"]
+samples = ["->", "->>", "-->", "--->", "<-->", "<--->", "<---->", "=>", "=>>", "==>", "===>", "<==>", "<===>", "<====>", "~>", "~~>", "-------->", ">=>", ">->"]
 desc = 'Enable ligation set that forms right-pointing arrows'

+[simple.arrow-lr]
+samples = ["<->", "<=>", "<~>"]
+desc = 'Enable ligation set that forms dual-pointing arrows'
+
 [simple.counter-arrow-l]
-ligGroup =  "counter-arrow-l"
 samples = ["-<<", "-<", "-<-", "=<<", "=<", "=<=", "<=<", "-<<", "-<", "-<-", "=<<", "=<", "=<="]
 desc = 'Enable ligation that produces left-pointing counter-arrows'

 [simple.counter-arrow-r]
-ligGroup =  "counter-arrow-r"
 samples = ["->-", ">-", ">>-", "=>=", ">=>", "->-", ">-", ">>-", ">>=", "=>="]
 desc = 'Enable ligation that produces right-pointing counter-arrows'

 [simple.trig]
-ligGroup =  "trig"
 samples = ["<|", "<|>", "|>"]
 desc = 'Enable ligation for `<|`, `|>` , `<||`, and other bar-and-angle-bracket symbols'

 [simple.eqeqeq]
-ligGroup =  "eqeqeq"
 samples = ["==="]
 sampleRank = 2
 desc = 'Enable special ligation for `===` with triple lines'

 [simple.eqeq]
-ligGroup =  "eqeq"
 samples = ["==", "==="]
 desc = 'Enable ligation for `==` and `===`'

 [simple.lteq]
-ligGroup =  "lteq"
 samples = ["<="]
 sampleRank = 2
 desc = 'Enable ligation for `<=` as less-than-or-equal sign'

 [simple.eqlt]
-ligGroup =  "eqlt"
 samples = ["=<"]
 sampleRank = 2
 desc = 'Enable ligation for `=<` as less-than-or-equal sign'

 [simple.gteq]
-ligGroup =  "gteq"
 samples = [">="]
 sampleRank = 2
 desc = 'Enable ligation for `>=` as greater-than-or-equal sign'

 [simple.lteq-separate]
-ligGroup =  "lteq-separate"
 samples = ["<="]
 sampleRank = 3
 desc = 'Display `<=` as separate shape'

 [simple.eqlt-separate]
-ligGroup =  "eqlt-separate"
 samples = ["=<"]
 sampleRank = 3
 desc = 'Display `=<` as separate shape'

 [simple.gteq-separate]
-ligGroup =  "gteq-separate"
 samples = [">="]
 sampleRank = 3
 desc = 'Display `>=` as separate shape'

 [simple.exeqeqeq]
-ligGroup =  "exeqeqeq"
 samples = ["!==="]
 sampleRank = 3
 desc = 'Enable special ligation for `!===` with triple lines'

 [simple.exeqeq]
-ligGroup =  "exeqeq"
 samples = ["!=="]
 sampleRank = 3
 desc = 'Enable special ligation for `!==` with triple lines'

 [simple.eqexeq]
-ligGroup =  "eqexeq"
 samples = ["=!="]
 desc = 'Enable special ligation for `=!=` with triple lines'

 [simple.eqexeq-dl]
-ligGroup =  "eqexeq-dl"
 samples = ["=!="]
 desc = 'Enable special ligation for `=!=` with double lines'

 [simple.exeq]
-ligGroup =  "exeq"
 samples = ["!=", "!==", "!==="]
 desc = 'Enable ligation for `!=` and `!==`'

 [simple.tildeeq]
-ligGroup =  "tildeeq"
 samples = ["~="]
 desc = 'Enable ligation for `~=` as inequality'

 [simple.eqslasheq]
-ligGroup = "eqslasheq"
 samples = ["=/="]
 sampleRank = 2
 desc = 'Enable special triple-line ligation for `=/=` as inequality'

 [simple.slasheq]
-ligGroup = "slasheq"
 samples = ["/=", "=/="]
 desc = 'Enable ligation for `/=` and `=/=` as inequality'

 [simple.ltgt-ne]
-ligGroup = "ltgt-ne"
 samples = ["<>"]
 sampleRank = 2
 desc = 'Enable ligation for `<>` as inequality'

 [simple.ltgt-diamond]
-ligGroup = "ltgt-diamond"
 samples = ["<>"]
 desc = 'Enable ligation for `<>` as diamond'

 [simple.ltgt-diamond-tag]
-ligGroup = "ltgt-diamond-tag"
 samples = ["<>"]
 sampleRank = 3
 desc = 'Enable ligation for `<>` as diamond-shaped empty HTML/XML tag'

 [simple.ltgt-slash-tag]
-ligGroup = "ltgt-slash-tag"
 samples = ["</", "/>", "</>"]
 sampleRank = 2
 desc = 'Enable ligation for `</`, `/>` and `</>`'

 [simple.brst]
-ligGroup = "brst"
 samples = ["(*", "*)", "(* comm *)"]
 desc = 'Center asterisk in `(*` and `*)`'

 [simple.slash-asterisk]
-ligGroup = "slash-asterisk"
 samples = ["/*", "*/", "/* comm */"]
 desc = 'Shift asterisk in `/*` and `*/`'

-[simple.plusplus]
-ligGroup =  "plusplus"
-samples = ["++", "+++"]
-desc = 'Enable ligation for `++` and further plus-chaining'
-
 [simple.kern-dotty]
-ligGroup =  "kern-dotty"
 samples = ["::", ":::"]
 desc = 'Move connecting dotty punctuations closer, like for `::`, `:::` and `...`'

 [simple.kern-bars]
-ligGroup =  "kern-bars"
 samples = ["||", "|||"]
 desc = 'Move consecutive bars closer, like for `||`, `|||` and `//`'

 [simple.logic]
-ligGroup =  "logic"
 samples = ["/\\", "\\/", "|-", "-|"]
 desc = 'Enable ligation for `/\` and `\/`'

 [simple.llgg]
-ligGroup =  "llgg"
 samples = ["<<", ">>", "<<<", ">>>"]
 desc = 'Enable ligation for `<<`, `>>` and other angle-bracket chaining'

 [simple.llggeq]
-ligGroup =  "llggeq"
 samples = ["<<=", ">>="]
 sampleRank = 2
 desc = 'Enable ligation for `<<=`, `>>=` as shift operator'

 [simple.html-comment]
-ligGroup =  "html-comment"
 samples = ["<!--", "<!---"]
 desc = 'Enable ligation for `<!--` and `<!---`'

 [simple.colon-greater-as-colon-arrow]
-ligGroup = "colon-greater"
 samples = [":>"]
 sampleRank = 2
 desc = 'Transform `:>` into `:` and a narrow arrow'

 [simple.brace-bar]
-ligGroup = 'brace-bar'
 samples = ["{|", "|}"]
 desc = 'Enable ligation for `{|` and `|}`'

 [simple.brack-bar]
-ligGroup = 'brack-bar'
 samples = ["[|", "|]"]
 desc = 'Enable ligation for `[|` and `|]`'

-[simple.connected-underscore]
-ligGroup = 'connected-underscore'
-samples = ["__", "___"]
-desc = 'Make contiguous underscores (like `__`) connected'
+# Connection
+[simple.tilde-tilde]
+samples = ["~~"]
+implies = ["tilde-tilde-tilde"]
+desc = 'Make 2 or more contiguous ASCII tildes (like `~~`, `~~~` and `~~~~`) connected as a wave line'

-[simple.connected-number-sign]
-ligGroup = 'connected-number-sign'
-samples = ["##", "###"]
-desc = 'Make contiguous number signs (like `##`) connected'
+[simple.tilde-tilde-tilde]
+samples = ["~~~", "~~~~"]
+desc = 'Make 3 or more contiguous ASCII tildes (like `~~~` and `~~~~`) connected as a wave line'

-[simple.connected-tilde-as-wave]
-ligGroup = 'connected-tilde-as-wave'
-samples = ["~~", "~~~"]
-desc = 'Make contiguous ASCII tildes (like `~~`) connected as a wave line'
+[simple.minus-minus]
+samples = ["--"]
+implies = ["minus-minus-minus"]
+desc = 'Make 2 or more contiguous hyphen-minuses (like `--`, `---` and `----`) connected as a straight solid line'

-[simple.connected-hyphen]
-ligGroup = 'connected-hyphen'
-samples = ["--", "---"]
-desc = 'Make contiguous hyphen-minuses (like `--`) connected as a straight solid line'
+[simple.minus-minus-minus]
+samples = ["---", "----"]
+desc = 'Make 3 or more contiguous hyphen-minuses (like `---` and `----`) connected as a straight solid line'
+
+[simple.plus-plus]
+samples = ["++"]
+implies = ["plus-plus-plus"]
+desc = 'Make 2 or more contiguous plus signs (like `++`, `+++` and `++++`) connected.'
+
+[simple.plus-plus-plus]
+samples = ["+++", "++++"]
+desc = 'Make 3 or more contiguous plus signs (like `+++` and `++++`) connected.'
+
+[simple.underscore-underscore]
+samples = ["__"]
+implies = ["underscore-underscore-underscore"]
+desc = 'Make 2 or more contiguous underscores (like `__`, `___` and `____`) connected'
+
+[simple.underscore-underscore-underscore]
+samples = ["___", "____"]
+desc = 'Make 3 or more contiguous underscores (like `___` and `____`) connected'
+
+[simple.hash-hash]
+samples = ["##"]
+implies = ["hash-hash-hash"]
+desc = 'Make 2 or more contiguous hash signs (number signs) (like `##`, `###` and `####`) connected'
+
+[simple.hash-hash-hash]
+samples = ["###", "####"]
+desc = 'Make 3 or more contiguous hash signs (number signs) (like `##` and `###`) connected'

 ###################################################################################################

@@ -300,28 +278,28 @@ buildup = [
 buildup = [ 'eqeq', 'exeq', 'lteq', 'gteq' ]

 [composite.--c-equality-inequality--]
-buildup = [	'eqeqeq', 'eqeq', 'exeqeq', 'exeq', 'lteq', 'gteq' ]
+buildup = [ 'eqeqeq', 'eqeq', 'exeqeq', 'exeq', 'lteq', 'gteq' ]

 [composite.--raku-equality-inequality--]
-buildup = [	'eqeqeq', 'eqeq', 'exeqeqeq', 'exeq', 'lteq', 'gteq' ]
+buildup = [ 'eqeqeq', 'eqeq', 'exeqeqeq', 'exeq', 'lteq', 'gteq' ]

 [composite.--ml-equality-inequality--]
-buildup = [	'eqeq', 'ltgt-ne', 'lteq', 'gteq' ]
+buildup = [ 'eqeq', 'ltgt-ne', 'lteq', 'gteq' ]

 [composite.--fstar-equality-inequality--]
-buildup = [	'eqeq', 'ltgt-ne', 'eqeqeq', 'eqexeq', 'lteq', 'gteq' ]
+buildup = [ 'eqeq', 'ltgt-ne', 'eqeqeq', 'eqexeq', 'lteq', 'gteq' ]

 [composite.--haskell-equality-inequality--]
-buildup = [	'eqeq', 'slasheq', 'lteq', 'gteq' ]
+buildup = [ 'eqeq', 'slasheq', 'lteq', 'gteq' ]

 [composite.--matlab-equality-inequality--]
-buildup = [	'eqeq', 'tildeeq', 'lteq', 'gteq' ]
+buildup = [ 'eqeq', 'tildeeq', 'lteq', 'gteq' ]

 [composite.--verilog-equality-inequality--]
-buildup = [	'eqeqeq', 'eqeq', 'exeqeq', 'exeq', 'lteq-separate', 'gteq-separate' ]
+buildup = [ 'eqeqeq', 'eqeq', 'exeqeq', 'exeq', 'lteq-separate', 'gteq-separate' ]

 [composite.--wolfram-equality-inequality--]
-buildup = [	'eqeq', 'exeq', 'eqexeq-dl', 'lteq', 'gteq' ]
+buildup = [ 'eqeq', 'exeq', 'eqexeq-dl', 'lteq', 'gteq' ]

 [composite.--erlang-equality-inequality--]
 buildup = [ 'eqeq', 'exeq', 'eqlt', 'gteq' ]
@@ -329,6 +307,15 @@ buildup = [ 'eqeq', 'exeq', 'eqlt', 'gteq' ]
 [composite.--default-kern--]
 buildup = [ 'kern-dotty', 'kern-bars' ]

+[composite.--default-chaining--]
+buildup = [ 'plus-plus-plus', 'minus-minus-minus', 'underscore-underscore-underscore', 'hash-hash-hash', 'tilde-tilde-tilde' ]
+
+[composite.--fast-chaining--]
+buildup = [ 'plus-plus', 'minus-minus', 'underscore-underscore', 'hash-hash', 'tilde-tilde' ]
+
+[composite.--c-like-chaining--]
+buildup = [ 'plus-plus', 'minus-minus-minus', 'underscore-underscore', 'hash-hash-hash', 'tilde-tilde-tilde' ]
+
 # This feature is on by default by many software
 [composite.default-calt]
 tag     = 'calt'
@@ -339,7 +326,8 @@ buildup = [
 	'--default-center-ops--',
 	'--default-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--default-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'html-comment',
 	'ltgt-diamond-tag',
 	'ltgt-slash-tag',
@@ -347,8 +335,6 @@ buildup = [
 	'slash-asterisk',
 	'llgg',
 	'llggeq',
-	'plusplus',
-	'connected-underscore'
 ]

 [composite.dlig]
@@ -360,7 +346,8 @@ buildup = [
 	'center-op-influence-dot',
 	'--default-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--fast-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'counter-arrow-l', 'counter-arrow-r',
 	'html-comment',
 	'ltgt-diamond-tag',
@@ -368,15 +355,10 @@ buildup = [
 	'trig',
 	'slash-asterisk',
 	'llgg',
-	'plusplus',
 	'logic',
 	'brst',
 	'brace-bar',
 	'brack-bar',
-	'connected-underscore',
-	'connected-number-sign',
-	'connected-tilde-as-wave',
-	'connected-hyphen'
 ]

 [composite.clike]
@@ -386,14 +368,13 @@ buildup = [
 	'--c-center-ops--',
 	'--c-equality-inequality--',
 	'--default-kern--',
-	'arrow-r',
+	'--c-like-chaining--',
+	'arrow-r', 'arrow-lr',
 	'ltgt-diamond',
 	'trig',
 	'slash-asterisk',
 	'llgg',
 	'llggeq',
-	'plusplus',
-	'connected-underscore'
 ]

 [composite.javascript]
@@ -403,7 +384,8 @@ buildup = [
 	'--c-center-ops--',
 	'--c-equality-inequality--',
 	'--default-kern--',
-	'arrow-r',
+	'--c-like-chaining--',
+	'arrow-r', 'arrow-lr',
 	'html-comment',
 	'ltgt-diamond-tag',
 	'ltgt-slash-tag',
@@ -411,8 +393,6 @@ buildup = [
 	'slash-asterisk',
 	'llgg',
 	'llggeq',
-	'plusplus',
-	'connected-underscore'
 ]

 [composite.php]
@@ -423,7 +403,8 @@ buildup = [
 	'center-op-influence-dot',
 	'--c-equality-inequality--',
 	'--default-kern--',
-	'arrow-r',
+	'--c-like-chaining--',
+	'arrow-r', 'arrow-lr',
 	'html-comment',
 	'ltgt-diamond-tag',
 	'ltgt-slash-tag',
@@ -431,8 +412,6 @@ buildup = [
 	'slash-asterisk',
 	'llgg',
 	'llggeq',
-	'plusplus',
-	'connected-underscore'
 ]

 [composite.julia]
@@ -443,11 +422,11 @@ buildup = [
 	'center-op-influence-dot',
 	'--c-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--default-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
 	'llgg',
 	'llggeq',
-	'plusplus',
 ]

 [composite.raku]
@@ -458,11 +437,11 @@ buildup = [
 	'center-op-influence-dot',
 	'--raku-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--c-like-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
 	'llgg',
 	'llggeq',
-	'plusplus',
 ]

 [composite.ml]
@@ -472,10 +451,10 @@ buildup = [
 	'--default-center-ops--',
 	'--ml-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--fast-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
 	'brst',
-	'plusplus',
 	'logic',
 ]

@@ -491,10 +470,10 @@ buildup = [
 	'--default-center-ops--',
 	'--fstar-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--fast-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
 	'brst',
-	'plusplus',
 	'logic',
 ]

@@ -506,19 +485,22 @@ buildup = [
 	'center-op-influence-dot',
 	'--haskell-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--fast-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'counter-arrow-l', 'counter-arrow-r',
 	'trig',
 	'llgg',
 	'ltgt-diamond',
-	'plusplus',
 	'logic',
 ]

 [composite.idris]
 tag     = 'IDRS'
 desc    = 'Idris'
-buildup = [ 'haskell' ]
+buildup = [
+	'haskell',
+	'brack-bar',
+]

 [composite.elm]
 tag     = 'ELMX'
@@ -537,13 +519,13 @@ buildup = [
 	'--default-center-ops--',
 	'--default-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--fast-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'counter-arrow-l', 'counter-arrow-r',
 	'trig',
 	'slash-asterisk',
 	'llgg',
 	'ltgt-diamond',
-	'plusplus',
 ]

 [composite.dafny]
@@ -555,13 +537,13 @@ buildup = [
 	'center-op-trigger-bar-r',
 	'--default-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--default-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'counter-arrow-l', 'counter-arrow-r',
 	'trig',
 	'slash-asterisk',
 	'llgg',
 	'ltgt-diamond',
-	'plusplus',
 ]

 [composite.coq]
@@ -572,11 +554,11 @@ buildup = [
 	'center-op-influence-dot',
 	'--ml-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--fast-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'counter-arrow-l', 'counter-arrow-r',
 	'trig',
 	'llgg',
-	'plusplus',
 	'logic',
 	'brst',
 ]
@@ -588,10 +570,10 @@ buildup = [
 	'--default-center-ops--',
 	'--matlab-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--default-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
 	'llgg',
-	'plusplus',
 ]

 [composite.verilog]
@@ -601,12 +583,12 @@ buildup = [
 	'--default-center-ops--',
 	'--verilog-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--default-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
 	'slash-asterisk',
 	'llgg',
 	'llggeq',
-	'plusplus',
 	'brst',
 ]

@@ -618,11 +600,11 @@ buildup = [
 	'--default-center-ops--',
 	'--wolfram-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--fast-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
 	'ltgt-diamond',
 	'brst',
-	'plusplus',
 	'logic',
 	'colon-greater-as-colon-arrow',
 ]
@@ -635,7 +617,8 @@ buildup = [
    	'--default-center-ops--',
 	'--erlang-equality-inequality--',
 	'--default-kern--',
-	'arrow-l', 'arrow-r',
+	'--default-chaining--',
+	'arrow-l', 'arrow-r', 'arrow-lr',
 	'trig',
-	'llgg'
+	'llgg',
 ]
